lib/YuiRestClient.pm

```diff
@@ -70,6 +75,21 @@ sub setup_libyui {
     connect_to_app;
 }
 
+sub setup_libyui_running_system {
+    add_suseconnect_product('sle-module-development-tools');
+    zypper_call('in libyui-rest-api');
+
+    my $port = get_var('YUI_PORT');
+    my $host = get_var('YUI_SERVER');
+    die "Cannot set libyui REST API server" unless $host;
```

Comment:
I guess we also should check if port is set or provide default value.

Reply:
IIRC this is set in `set_libyui_backend_vars` where set a random one or die.

Reply:
For this case, we then can delegate it to that piece of code which sets those, so not validate `$host` here. So I would say either we validate both, as main.pm call here is not visible, or none and only log those, so it's still easy to spot when failure happened.